app/connectors/AmlsConnector.scala

```diff
@@ -193,6 +193,14 @@ class AmlsConnector @Inject()(val httpPost: WSHttp,
     httpPost.POST[DeRegisterSubscriptionRequest, DeRegisterSubscriptionResponse](postUrl, request)
```

Comment:
above todo is not needed as line 186 explains what to do here

Reply:
Done :)